redisinsight/ui/src/pages/browser/components/key-details-header/KeyDetailsHeader.tsx

```diff
@@ -307,7 +309,10 @@ const KeyDetailsHeader = ({
 
   const Actions = (width: number) => {
     const isEditable = !isStringCompressed && isFormatEditable(viewFormatProp)
+    const isStringEditable = keyType === KeyTypes.String ? keyValue?.data?.length === length : true
```

Comment:
You created `isFullStringLoaded` I think you should use it instead of `keyValue?.data?.length === length`